watsononlinestore/watson_online_store.py

```diff
@@ -448,35 +547,35 @@ def get_product_url(entry, data_source):
             if 'html' in entry:
                 html = entry['html']
 
-                if data_source == "amazon":
-                    # For amazon data, the product URL is stored in a
-                    # "<a href" tag located at the end of the html doc.
-                    href_tag = "<a href="
+                if data_source == AMAZON:
+                    # For amazon data, the product URL is stored in an
+                    # href tag located at the end of the html doc.
+                    HREF_TAG = "<a href="
                     # Search from bottom of the doc.
-                    sidx = html.rfind(href_tag)
+                    sidx = html.rfind(HREF_TAG)
                     if sidx > 0:
-                        sidx += len(href_tag)
+                        sidx += len(HREF_TAG)
                         eidx = html.find('>', sidx, len(html))
                         if eidx > 0:
                             product_url = html[sidx+1:eidx-1]
-                elif data_source == "ibm_store":
+                elif data_source == IBM:
                     # For IBM store data, the product URL requires a
                     # product ID. The product ID can be found by searching
                     # the html doc for "/ProductDetail.aspx?pid=<PID>".
                     # The product URL can then be built by appending
                     # this string to:
                     # ""http://www.logostore-globalid.us".
-                    url_start = "http://www.logostore-globalid.us"
-                    href_tag = "/ProductDetail.aspx?pid="
-                    sidx = html.find(href_tag)
+                    URL_START = "http://www.logostore-globalid.us"
+                    HREF_TAG = "/ProductDetail.aspx?pid="
+                    sidx = html.find(HREF_TAG)
                     if sidx > 0:
-                        sidx += len(href_tag)
+                        sidx += len(HREF_TAG)
                         product_id = html[sidx:sidx+6]
-                        product_url = url_start + href_tag + product_id
+                        product_url = URL_START + URL_START + product_id
 
             return product_url
```

Comment:
Need an else? How to ensure product_url is set?
Also, probably not now, but seems like eventually store classes with the impl of the few methods might be nicer than the if ibm, elif az, ... repetition.